elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -40,17 +40,24 @@ $variables: (
   background-color: #{pfe-local(BackgroundColor)};
   color: #{pfe-local(Color)};
 
-  // Exposing this for customization
-  text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
-  font-weight: #{pfe-var(font-weight--normal)};
-
   cursor: pointer;
   text-align: #{pfe-local($cssvar: TextAlign, $region: tab, $fallback: center)};
   
-  ::slotted(*) {
-    color: #{pfe-local(Color)} !important;
+  // Exposing this for customization
+  %typography {
+    text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
+    font-family: #{pfe-var(font-family)};
+    font-weight: #{pfe-var(font-weight--normal)};
+    font-size: #{pfe-var(font-size)};
```

Comment:
One test case I was trying was attempting to adjust the font size of all the tab headings on the page, but use of the variable. If i set it on the body tag it works: `body {--pfe-theme--font-size: 29px} but if I set it on section or pfe-tabs or pfe-tab, it does not work. 

Is that because shadow-root can only inherit from global scope? if that's the case, I think we will need unique variables like `--pfe-tab__header--font-size`.

Reply:
It's because some of the tabs don't have htags and I attached the styles only to htags.  I should both abstract that a layer higher and ensure an htag exists in the shadow DOM.

Reply:
Fixed this issue now